images/nginx/rootfs/build.sh

```diff
@@ -421,6 +421,11 @@ sh build.sh
 make
 make install
 
+# Copy over modsecurity.conf
```

Comment:
```
-------------------------------------------------------------------------------
W1104 04:52:36.972626       9 queue.go:130] requeuing configmap-change, err 
-------------------------------------------------------------------------------
Error: exit status 1
2018/11/04 04:52:36 [notice] 1600#1600: ModSecurity-nginx v1.0.0
2018/11/04 04:52:36 [emerg] 1600#1600: "modsecurity_rules_file" directive Rules error. File: /etc/nginx/modsecurity/modsecurity.conf. Line: 236. Column: 17. Failed to locate the unicode map file from: unicode.mapping Looking at: 'unicode.mapping', 'unicode.mapping', '/etc/nginx/modsecurity/unicode.mapping', '/etc/nginx/modsecurity/unicode.mapping'.  in /tmp/nginx-cfg215230822:325
nginx: [emerg] "modsecurity_rules_file" directive Rules error. File: /etc/nginx/modsecurity/modsecurity.conf. Line: 236. Column: 17. Failed to locate the unicode map file from: unicode.mapping Looking at: 'unicode.mapping', 'unicode.mapping', '/etc/nginx/modsecurity/unicode.mapping', '/etc/nginx/modsecurity/unicode.mapping'.  in /tmp/nginx-cfg215230822:325
nginx: configuration file /tmp/nginx-cfg215230822 test failed
-------------------------------------------------------------------------------
```
I think we still need these changes in the image.

@aledbf ^^

Reply:
@diazjf please remove the second commit. The image is updated with the latest version of the modules and the mapping file in #3357

Also, if a change is required in the nginx image, that needs to be done in a separate PR.

Reply:
Done!